frozenlist/_frozenlist.pyx

```diff
@@ -71,20 +75,20 @@ cdef class FrozenList:
         if op == 5:  # =>
             return list(self) >= other
 
-    def insert(self, pos, item):
+    def insert(self, *args):
```

Comment:
This is probably a bad idea.

Reply:
I don't understand what you're trying to achieve but here's a typical signature: https://docs.python.org/3/library/stdtypes.html#sequence.insert
We shouldn't have differing signatures when we mimic interfaces.